### PR TITLE
ticdc: support to replication table without valid index

### PR DESCRIPTION
**Pull request by amyangfei**
### What is changed, added or deleted? (Required)

update cdc replication restriction, add instruction about how to replicate tables without valid index

### Which TiDB version(s) do your changes apply to? (Required)

- [x] master (the latest development version)
- [x] v4.0 (TiDB 4.0 versions)
- [ ] v3.1 (TiDB 3.1 versions)
- [ ] v3.0 (TiDB 3.0 versions)
- [ ] v2.1 (TiDB 2.1 versions)

### What is the related PR or file link(s)?

- This PR is translated from:
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Have version specific changes 
- [ ] Might cause conflicts

### CODE REVIEW

**Comment by amyangfei** [comment]
PTAL @leoppro @TomShawn

File: ticdc/manage-ticdc.md [hunk 1]
```diff
@@ -776,3 +776,18 @@ enable-old-value = true
 ```
 
 开启该特性后，TiCDC Open Protocol 的输出格式参考 [TiCDC 开放数据协议 - Row Changed Event](/ticdc/ticdc-open-protocol.md#row-changed-event)，使用 MySQL sink 时也会自动支持的 TiDB 4.0 新 Collation 特性。
+
+## 同步没有有效索引的表
+
+从 v4.0.8 开始，TiCDC 支持修改任务配置同步没有有效索引的表。若要开启该特性，需要在 changefeed 的配置文件的根级别指定配置：
```
Comment (TomShawn) [inline ticdc/manage-ticdc.md:782]:
```suggestion
从 v4.0.8 开始，TiCDC 支持通过修改任务配置来同步没有有效索引的表。若要开启该特性，需要在 `changefeed` 配置文件的根级别进行如下指定：
```

File: ticdc/manage-ticdc.md [hunk 2]
```diff
@@ -776,3 +776,18 @@ enable-old-value = true
...
+{{< copyable "" >}}
+
+```toml
+enable-old-value = true
+force-replicate = true
+```
+
+> **警告：**
+>
+> 没有有效索引的表，INSERT, REPLACE 等操作不具备可重入性，因此会有数据冗余的风险。CDC 在同步过程中只保证数据至少分发一次，因此开启该特性同步没有有效索引的表，数据冗余一定会出现。如果不能接受数据冗余，建议增加有效索引，譬如增加使用 AUTO RANDOM 的 PRIMARY KEY 列。
```
Comment (TomShawn) [inline ticdc/manage-ticdc.md:793]:
```suggestion
> 对于没有有效索引的表，`INSERT` 和 `REPLACE` 等操作不具备可重入性，因此会有数据冗余的风险。TiCDC 在同步过程中只保证数据至少分发一次，因此开启该特性同步没有有效索引的表，一定会导致数据冗余出现。如果不能接受数据冗余，建议增加有效索引，譬如为主键列添加 `AUTO RANDOM` 属性。
```

File: ticdc/ticdc-overview.md [hunk 3]
```diff
@@ -73,6 +73,8 @@ TiCDC 只能同步至少存在一个**有效索引**的表，**有效索引**的
     - 索引中每一列在表结构中明确定义非空 (`NOT NULL`)。
     - 索引中不存在虚拟生成列 (`VIRTUAL GENERATED COLUMNS`)。
 
+TiCDC 从 4.0.8 版本开始，支持修改任务配置同步没有**有效索引**的表，但在数据一致性的保证上有所减弱，具体使用方法和注意事项参考[同步没有有效索引的表](/ticdc/manage-ticdc.md#同步没有有效索引的表)。
```
Comment (TomShawn) [inline ticdc/ticdc-overview.md:76]:
```suggestion
TiCDC 从 4.0.8 版本开始，可通过修改任务配置来同步**没有有效索引**的表，但在数据一致性的保证上有所减弱。具体使用方法和注意事项参考[同步没有有效索引的表](/ticdc/manage-ticdc.md#同步没有有效索引的表)。
```

**Review by TomShawn — COMMENTED** [review]
Rest LGTM

File: ticdc/manage-ticdc.md [hunk 4]
```diff
@@ -776,3 +776,18 @@ enable-old-value = true
...
+{{< copyable "" >}}
+
+```toml
+enable-old-value = true
+force-replicate = true
+```
+
+> **警告：**
+>
+> 对于没有有效索引的表，`INSERT` 和 `REPLACE` 等操作不具备可重入性，因此会有数据冗余的风险。TiCDC 在同步过程中只保证数据至少分发一次，因此开启该特性同步没有有效索引的表，一定会导致数据冗余出现。如果不能接受数据冗余，建议增加有效索引，譬如为主键列添加 `AUTO RANDOM` 属性。
```
Comment (amyangfei) [inline ticdc/manage-ticdc.md:793]:
```suggestion
> 对于没有有效索引的表，`INSERT` 和 `REPLACE` 等操作不具备可重入性，因此会有数据冗余的风险。TiCDC 在同步过程中只保证数据至少分发一次，因此开启该特性同步没有有效索引的表，一定会导致数据冗余出现。如果不能接受数据冗余，建议增加有效索引，譬如增加具有 `AUTO RANDOM` 属性的主键列。
```

**Comment by TomShawn** [comment]
LGTM

**Comment by leoppro** [comment]
LGTM

**Review by TomShawn — APPROVED** [review]

**Pull request merged**